maintainers/maintainer-list.nix

```diff
@@ -6424,6 +6424,10 @@
     githubId = 1447245;
     name = "Robin Gloster";
   };
+  gm6k = {
+    email = "nix@quidecco.pl";
```

Comment:
Could you add your github nickname also? (could be convenient for automation and other to find you on github)

Reply:
hehe, I unrderstand!